packages/backend-test-utils/src/next/wiring/TestBackend.ts

```diff
@@ -101,4 +123,32 @@ export async function startTestBackend<
   }
 
   await backend.start();
+
+  return backend;
 }
+
+let registered = false;
+function registerTestHooks() {
+  if (typeof afterEach !== 'function') {
```

Comment:
It's a bit funny that we check afterEach but call afterAll :) It'll work in practice, buuuut

Reply:
doh